utils/api/caseStatus.ts

```diff
@@ -0,0 +1,8 @@
+import useSWR, { SWRResponse } from 'swr';
+
+import type { PersonCaseStatus, ErrorAPI } from 'types';
+
+export const GetCaseStatus = (
```

Comment:
convention: should be `getCaseStatus`

Reply:
issue with my linter. Jack is having a look.